### PR TITLE
Addition-text-blok-with-inner-developers-reference

### PR DESCRIPTION
**Pull request by ValDekh**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/app/router/Routes.tsx
Comment (ormykhalyshyn) [inline src/app/router/Routes.tsx:1]: You need to fix this.
All admin components must be protected, and the code must be without such comments.

File: src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx
Comment (ormykhalyshyn) [inline src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx:1]: Change this import TimelineApi from '../../../app/api/timeline/timeline.api';
To like this import TimelineItem from '@/models/timeline/chronology.model';

File: src/features/AdminPage/NewStreetcode/TextBlock/TextBlock.component.tsx
Comment (ormykhalyshyn) [inline src/features/AdminPage/NewStreetcode/TextBlock/TextBlock.component.tsx:1]: change syntaxis for import

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/TextForm.component.tsx
Comment (ormykhalyshyn) [inline src/features/AdminPage/NewStreetcode/TextBlock/TextForm/TextForm.component.tsx:1]: same import

File: src/features/StreetcodePage/TextBlock/TextBlock.component.tsx
Comment (ormykhalyshyn) [inline src/features/StreetcodePage/TextBlock/TextBlock.component.tsx:1]: delete comments

**Review by ormykhalyshyn — CHANGES_REQUESTED** [review]
small fixes, but overall good job)

File: src/app/router/Routes.tsx
Comment (ValDekh) [reply src/app/router/Routes.tsx:1]: Done

File: src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx:1]: Done

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx
[the remark this answers is not in the document]
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx:1]: done

File: src/features/AdminPage/NewStreetcode/TextBlock/TextBlock.component.tsx
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/TextBlock/TextBlock.component.tsx:1]: this imports work correct, I hadn't changed it previously

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/AdditionTextBlock/AdditionalTextBlockAdminForm.component.tsx
[the remark this answers is not in the document]
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/TextBlock/TextForm/AdditionTextBlock/AdditionalTextBlockAdminForm.component.tsx:1]: done

File: src/features/AdminPage/NewStreetcode/TextBlock/TextBlock.component.tsx
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/TextBlock/TextBlock.component.tsx:1]: Done

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/TextForm.component.tsx
Comment (ValDekh) [reply src/features/AdminPage/NewStreetcode/TextBlock/TextForm/TextForm.component.tsx:1]: Done

File: src/features/StreetcodePage/Streetcode.component.tsx
[the remark this answers is not in the document]
Comment (ValDekh) [reply src/features/StreetcodePage/Streetcode.component.tsx:1]: done

File: src/features/StreetcodePage/TextBlock/TextBlock.component.tsx
Comment (ValDekh) [reply src/features/StreetcodePage/TextBlock/TextBlock.component.tsx:1]: Done

File: src/models/streetcode/text-contents.model.ts
[the remark this answers is not in the document]
Comment (ValDekh) [reply src/models/streetcode/text-contents.model.ts:1]: Done

File: src/models/streetcode/streetcode-types.model.ts
[the remark this answers is not in the document]
Comment (ValDekh) [reply src/models/streetcode/streetcode-types.model.ts:1]: Done

**Review by NadiaKishchuk — APPROVED** [review]

File: src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx [hunk 1]
```diff
@@ -285,7 +288,7 @@ const NewStreetcode = () => {
             <PageBar />
             <ConfigProvider locale={ukUA}>
                 <div className="adminContainer">
-                    {/* <StreetCodeBlock /> */}
```
Comment (Dobriyr) [inline src/features/AdminPage/NewStreetcode/MainNewStreetcode.component.tsx:288]: this comment should be left, we need to remember about this block in the future

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/AdditionTextBlock/AdditionalTextBlockAdminForm.component.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-restricted-imports */
+/* eslint-disable react/jsx-props-no-multi-spaces */
```
Comment (Dobriyr) [inline src/features/AdminPage/NewStreetcode/TextBlock/TextForm/AdditionTextBlock/AdditionalTextBlockAdminForm.component.tsx:2]: missed comment

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx [hunk 3]
```diff
@@ -1,20 +1,21 @@
+/* eslint-disable react/jsx-props-no-multi-spaces */
```
Comment (Dobriyr) [inline src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx:1]: 1 more

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx [hunk 4]
```diff
@@ -84,22 +86,22 @@ const TextEditor = ({ inputInfo, setInputInfo } : Props) => {
...
+                >
+                    {getTermArray.map(
+                        (t) => <Select.Option key={t.id} value={t.title}>{t.title}</Select.Option>,
+                    )}
+                </AutoComplete>
             </Form.Item>
-            
-            <div className='display-flex-row'>
+
+            <div className="display-flex-row">
```
Comment (Dobriyr) [inline src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx:102]: the style of the code is scattered
it is worth using the same quotation marks for all strings

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx [hunk 5]
```diff
@@ -84,22 +86,22 @@ const TextEditor = ({ inputInfo, setInputInfo } : Props) => {
...
                 <Button
-                    className = 'streetcode-custom-button button-margin-vertical button-margin-right'
+                    className="streetcode-custom-button button-margin-vertical button-margin-right"
```
Comment (Dobriyr) [inline src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx:104]: same here

File: src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx [hunk 6]
```diff
@@ -108,12 +110,12 @@ const TextEditor = ({ inputInfo, setInputInfo } : Props) => {
                 <Button
                     onClick={handleDeleteRelatedWord}
                     disabled={selected === '' || term === undefined}
-                    className = 'streetcode-custom-button button-margin-vertical'
+                    className="streetcode-custom-button button-margin-vertical"
```
Comment (Dobriyr) [inline src/features/AdminPage/NewStreetcode/TextBlock/TextForm/Editors/TextEditor.component.tsx:113]: and here)

File: src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+    font-style: italic;
+    display: flex;
+    justify-content:flex-end;
+
+    p {
+        font-family: 'Closer Text' !important;
+
+        a {
+            font-weight: 350;
+            color: #891F16;
```
Comment (Dobriyr) [inline src/features/StreetcodePage/TextBlock/AdditionalTextBlock/AdditionalTextBlock.styles.scss:25]: should use or create color from our assets

**Review by Dobriyr — CHANGES_REQUESTED** [review]
great work, but there is a note about the code style

**Pull request closed without merging**